doc/api/modules.md

```diff
@@ -235,7 +235,7 @@ the `require.resolve()` function.
 Putting together all of the above, here is the high-level algorithm
 in pseudocode of what `require()` does:
 
-<pre>
+```
```

Comment:
````suggestion
```text
````

Reply:
That shouldn't matter. By default typeless code blocks are plain text.

Reply:
But apparently it matters for our linter 🤷